### ISSUE TITLE
splunk.app_paths.idxc not used for configuring system configurations (_cluster)

### ISSUE THREAD

**Issue by mshensg** [splunk/splunk-ansible#776]
although we are allowed to use splunk.app_paths.idxc to point the idx cluster apps to manager-apps instead of master-apps. the cluster master/manager playbooks (/roles/splunk_cluster_master/tasks/configure_indexes.yml or /roles/splunk_cluster_master/tasks/configure_server.yml) will not use this information, they are hard coded to put configurations into master-apps. then the system will not be able to work properly when we want to correct this language bias. Please help work on it to make things consistent. thanks.

**Comment by adityapinglesf** [comment]
yes, working on fixing the language bias. Will update the status and this issue soon.
